ready_recipes/tifffile/meta.yaml

```diff
@@ -24,3 +24,8 @@ about:
   home: https://github.com/blink1073/tifffile
   license:  BSD License
   summary: 'Read and write image data from and to TIFF files.'
+
+extra:
+  recipe-maintainers:
+    - blink1073
```

Comment:
Thought as much, but wanted to give you a heads up. You should be added (automatically) to a conda-forge/tifffile team shortly.